mavros/scripts/mavparam

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Changing default python is not case of that PR. Please revert that.

Anyway scripts should work under py3 too, but that require to build ros with experimental py3 support.